core/pom.xml

```diff
@@ -39,19 +40,14 @@
             <version>3.3.0.Final</version>
         </dependency>
         <dependency>
-            <groupId>org.jboss.spec.javax.batch</groupId>
-            <artifactId>jboss-batch-api_1.0_spec</artifactId>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
             <groupId>javax.inject</groupId>
             <artifactId>javax.inject</artifactId>
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>javax.enterprise</groupId>
-            <artifactId>cdi-api</artifactId>
-            <scope>provided</scope>
+            <groupId>javax.batch</groupId>
+            <artifactId>javax.batch-api</artifactId>
+            <version>${javax.batch}</version>
```

Comment:
Why isn't this dependency marked as "provided" as `jboss-batch-api_1.0_spec` was?
I think it would make sense, especially given we used the "provided" scope for `javax.inject`.
Note that the scope of the dependency can be omitted here and moved up to the parent's dependency management (same for `javax.inject`).